operator/config/crd/bases/execution.securecodebox.io_scancompletionhooks.yaml

```diff
@@ -174,6 +178,12 @@ spec:
                       type: string
                   type: object
                 type: array
+              priority:
+                default: 0
+                description: Defines the priority of the hook. Higher priority hooks
```

Comment:
> Defines the priority of the hook.

I think this sentence is not helpful. Remove it?

Also mention the order of execution within priority classes. (Or reference more detailed docs, but in this case, I would mention this straight away.)